ibtk/include/ibtk/ibtk_utilities.h

```diff
@@ -268,6 +268,20 @@ smooth_heaviside(const double& phi, const double& alpha)
     return Hphi;
 }
 
+/*!
+ * Smooth delta function.
```

Comment:
I think `smooth_delta` represents derivative of `smooth_heaviside` function that is also defined in this file. The name makes this connection explicit. `cosine_kernel` will hide.

Reply:
IDK, if you ask me, we should do everything we can to discourage people from using the cosine delta function. 😉

Reply:
Its purpose is different here. It is not used for interpolation or as such, but to define a smooth Heaviside function to mark domain of a phase. The integral of Heaviside function gives total volume of the phase while the derivative of Heaviside function gives the surface area of the phase. Not at all related to IB...

Reply:
It might be worth trying to come up with some way to make this clear in the function name --- maybe by attaching these functions to a different namespace or class.